migrations/add-created-at-to-migrations-table.js

```diff
@@ -0,0 +1,22 @@
+const debug = require('debug')('knex-migrator:migration');
+
+/**
+ * @description Private helper to migrate the migrations table. It adds the created_at column to the migrations table.
+ * @returns {*}
+ */
+module.exports.up = function (connection) {
+    debug('Ensure created_at column.');
+
+    return connection.schema.hasTable('migrations')
+        .then(function (exists) {
+            if (exists) {
+                return connection.schema.alterTable('migrations', function (table) {
+                    const now = this.client.raw('CURRENT_TIMESTAMP');
+                    table.timestamp('created_at').notNullable().defaultTo(now);
```

Comment:
Was thinking about the implications of the `now` default and didn't come up with anything better. Trying to figure out more precise timestamps for past migrations would be a rabbit hole :tophat: